boards/arm/stm32f746g_disco/doc/stm32f746g_disco.rst

```diff
@@ -0,0 +1,200 @@
+.. _stm32f746g_disco_board:
+
+ST STM32F746G Discovery
+#######################
+
+Overview
+********
+
+The discovery kit enables a wide diversity of applications taking benefit from audio, multi-sensor support, graphics, security, video and high-speed connectivity features.
```

Comment:
Preference is for lines to be <80 characters

Change the ending to read:

   ... security, video, and high-speed connectivity features.  Important board features include: